src/main/resources/templates/checkin-form.html

```diff
@@ -1,63 +1,87 @@
-    Return
-  </a>
-
-<p class="h4 mb-4">Check in to Room</p>
-
-<!-- todo: route this service -->
-<form action="#" th:action="@{/Rooms/save}" th:object="${Room}" method="POST">
-
-<!-- invisible form as needed to update info -->
-<input type="hidden" th:field="*{id}"/>
-
-<input type="text" th:field="*{firstName}"
-class="form-control mb4 col-4" placeholder="First Name">
-
-<input type="text" th:field="*{lastName}"
-class="form-control mb4 col-4" placeholder="Last Name">
-
-<input type="text" th:field="*{roomNumber}"
-class="form-control mb4 col-4" placeholder="Room number (eg: 302)">
-
-<input type="text" th:field="*{checkinDate}"
-class="form-control mb4 col-4" placeholder="April 12">
-
-<input type="text" th:field="*{checkoutDate}"
-class="form-control mb4 col-4" placeholder="April 16">
-
-
-<button type="submit" class="btn btn-danger col-2">Register</button>
-
-</form>
-
-
-
-</div>
-
-
-
-
-</body>
-
-</html>
\ No newline at end of file
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-uWxY/CJNBR+1zjPWmfnSnVxwRheevXITnMqoEIeG1LJrdI0GlVs/9cVSyPYXdcSF"
+      crossorigin="anonymous"
+    />
+
+    <title>Check in</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <h4>Check in</h4>
+
+      <!-- todo: route this to home page -->
+      <a
+        class="btn btn-primary"
+        data-bs-toggle="collapse"
+        th:href="@{/home}"
+        role="button"
+        aria-expanded="false"
+        aria-controls="collapseExample"
+      >
+        Return
+      </a>
+
+      <p class="h4 mb-4">Check in to Room</p>
+
+      <!-- todo: route this service -->
+      <form
+        action="#"
+        th:action="@{/Rooms/save}"
+        th:object="${Room}"
+        method="POST"
+      >
+        <!-- invisible form as needed to update info -->
+        <input type="hidden" th:field="*{id}" />
+
+        <input
+          type="text"
+          th:field="*{firstName}"
+          class="form-control mb4 col-4"
+          placeholder="First Name"
+        />
+
+        <input
+          type="text"
+          th:field="*{lastName}"
+          class="form-control mb4 col-4"
+          placeholder="Last Name"
+        />
+
+        <input
+          type="text"
+          th:field="*{roomNumber}"
+          class="form-control mb4 col-4"
+          placeholder="Room number (eg: 302)"
+        />
+
+        <input
+          type="text"
+          th:field="*{checkinDate}"
+          class="form-control mb4 col-4"
+          placeholder="April 12"
+        />
+
+        <input
+          type="text"
+          th:field="*{checkoutDate}"
+          class="form-control mb4 col-4"
+          placeholder="April 16"
+        />
+
+        <button type="submit" class="btn btn-danger col-2">Register</button>
+      </form>
+    </div>
+  </body>
+</html>
```

Comment:
w3schools HTML Style guide encourages formatting that avoids long code lines, "When using an HTML editor, it is NOT convenient to scroll right and left to read the HTML code."
I formatted the html to shorten the longer input tags.

https://www.w3schools.com/html/html5_syntax.asp